src/c.rs

```diff
@@ -41,17 +41,6 @@ macro_rules! define_metrics_tests {
       $expected_align_factor:expr ) =>
     {
         #[cfg(test)]
-        extern {
```

Comment:
Leaving this in didn't work when adding `unsigned_long` because it uses the same size and alignment C globals, which then would get redefined.